src/jupyter-datascience-notebooks/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,52 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers-community/templates/tree/main/src/jupyter-datascience-notebooks
+{
+  "name": "Jupyter Data Science Notebooks",
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  "features": {
+    "ghcr.io/devcontainers/features/common-utils:2": {
+      "installZsh": "true",
+      "username": "jovyan",
+      "userUid": "1000",
+      "userGid": "1000",
+      "upgradePackages": "true"
+    }
+  },
+  // Use base images default CMD.
+  "overrideCommand": false,
+  // Forward Jupyter port locally, mark required.
+  "forwardPorts": [8888],
+  "portsAttributes": {
+    "8888": {
+      "label": "Jupyter",
+      "requireLocalPort": true,
+      "onAutoForward": "ignore"
+    }
+  },
```

Comment:
These settings are probably worth mentioning in the documentation, as they may not be necessary when using VS Code (I've just copied them for now)